src/bundle/Resources/public/scss/_general.scss

```diff
@@ -0,0 +1,225 @@
+.ibexa-field-edit--ezrichtext {
+    .ibexa-data-source {
+        border: calculateRem(1px) solid $ibexa-color-dark-200;
+        border-radius: $ibexa-border-radius;
+
+        .ck-content {
+            min-height: calculateRem(100px);
+            padding: 0 calculateRem(16px);
+        }
+
+        .ck-widget {
+            transition: none;
+            border-radius: $ibexa-border-radius;
+
+            &:hover,
+            &.table:hover {
+                outline-color: $ibexa-color-primary;
+                outline: none;
+                border: calculateRem(1px) solid $ibexa-color-primary;
+            }
+
+            &.ck-widget_with-selection-handle .ck-widget__selection-handle,
+            &.ck-widget_with-selection-handle:hover .ck-widget__selection-handle {
+                left: calculateRem(-1px);
+            }
+
+            &.ck-widget_with-selection-handle:hover .ck-widget__selection-handle,
+            &:not(.ck-widget_selected) > .ck-widget__type-around > .ck-widget__type-around__button {
+                background: $ibexa-color-primary;
+            }
+
+            &.ck-widget_selected,
+            &.ck-widget_selected:hover {
+                outline-color: $ibexa-color-dark;
+                outline: none;
+                border: calculateRem(1px) solid $ibexa-color-dark;
+            }
+
+            &.ck-widget_with-selection-handle.ck-widget_selected .ck-widget__selection-handle,
+            &.ck-widget_with-selection-handle.ck-widget_selected:hover .ck-widget__selection-handle,
+            &.ck-widget_selected > .ck-widget__type-around > .ck-widget__type-around__button {
+                background: $ibexa-color-dark;
+            }
+
+            &.table,
+            table {
```

Comment:
I think we should avoid styling on tags

Reply:
We should but this overrides CKEditor styles